src/Microsoft.DotNet.Interactive/KernelExtensions.cs

```diff
@@ -93,15 +93,29 @@ public static TKernel UseImportMagicCommand<TKernel>(this TKernel kernel)
         var command = new Command("#!import", "Imports and runs another notebook.");
         command.AddArgument(new Argument<FileInfo>("notebookFile").ExistingOnly());
         command.Handler = CommandHandler.Create(
-            async (FileInfo notebookFile, KernelInvocationContext _) =>
+            async (FileInfo notebookFile, KernelInvocationContext context) =>
             {
                 var document = await InteractiveDocument.LoadAsync(
                     notebookFile,
                     CreateKernelInfos(kernel.RootKernel as CompositeKernel));
 
                 foreach (var element in document.Elements)
                 {
-                    await kernel.RootKernel.SendAsync(new SubmitCode(element.Contents, element.KernelName));
+                    switch (element.KernelName?.ToLowerInvariant())
+                    {
+                        case "markdown":
+                            var @event = new DisplayedValueProduced(element.Contents, context.Command, new[]
+                            {
+                                new FormattedValue("text/markdown", element.Contents),
+                                new FormattedValue(PlainTextFormatter.MimeType, element.Contents)
```

Comment:
I don't see the plain text output being tested, and I'm not convinced this is the correct value for it. I could make the argument that the plain text version should remove the formatting characters from the Markdown, e.g. convert it to HTML and then grab the inner text from the HTML.

Reply:
Makes sense